aiu_fms_testing_utils/utils/aiu_setup.py

```diff
@@ -57,6 +59,8 @@ def aiu_setup(rank=0, world_size=1, local_rank=0, local_size=1, verbose=False):
 def aiu_dist_setup(rank, world_size, local_rank=-0, local_size=-1, verbose=False):
     if local_rank < 0:
         local_rank = rank
+
+    # FIXME: local_size not in use ?
```

Comment:
maybe it's no longer needed? if you can't find any reference to it feel free to delete